python_modules/dagster-graphql/dagster_graphql/schema/solids.py

```diff
@@ -382,6 +383,23 @@ def resolve_output_definitions(self, _graphene_info):
             for output_def_snap in self._solid_def_snap.output_def_snaps
         ]
 
+    def resolve_asset_nodes(self, graphene_info):
+        # NOTE: This is a temporary hack. We really should prob be resolving solids against the repo
+        # rather than pipeline, that way we would not have to refetch the repo here here in order to
```

Comment:
this repo fetch is generally not that expensive, since we hold the loaded repos in memory.